templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
+TARGET_PLIST="${TARGET_BUILD_DIR}/${INFOPLIST_PATH}"
+DSYM_PLIST="${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Info.plist"
+
+for PLIST in $TARGET_PLIST $DSYM_PLIST
+do
+  if [ -f "$PLIST" ] ; then
```

Comment:
You're mixing syntax here:

```sh
for x in y
do

# vs

for x in y; do

# is the same as

if [ ... ]
then

# vs

if [ ... ]; then
```

We prefer the semi-colon versions, but you should at least be consistent.